apps/api/mix.exs

```diff
@@ -1,10 +1,45 @@
 defmodule Api.MixProject do
   use Mix.Project
 
+  defp version do
+    case :file.consult('hex_metadata.config') do
```

Comment:
If we're going to go this route, we should at least ditch this clause, since there should never be a hex_metadata.config file in our setup (we never publish to hex)